CHANGELOG.md

```diff
@@ -12,6 +15,10 @@ Contributors:
 ### Under the hood
 - Switch to full reparse on partial parsing exceptions. Log and report exception information. ([#3725](https://github.com/dbt-labs/dbt/issues/3725), [#3733](https://github.com/dbt-labs/dbt/pull/3733))
 - Check for existence of test node when removing. ([#3711](https://github.com/dbt-labs/dbt/issues/3711), [#3750](https://github.com/dbt-labs/dbt/pull/3750))
+- Better error handling for BigQuery job labels that are too long. [#3703](https://github.com/dbt-labs/dbt/pull/3703)
```

Comment:
Is this right? We already released this one so I'm guessing there shouldn't be anymore additions here

Reply:
I don't know how that got in. Fixing up the CHANGELOG gets so confusing. Looking forward to the new tool... I've removed it.